pvlib/snowcoverage.py

```diff
@@ -0,0 +1,229 @@
+"""
+The ``snowcoverage`` module contains functions that model snow coverage on
+solar modules. The model was first proposed in Marion et al. 2013 and was
+later validated and implemented into NREL's SAM.
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def snow_slide_amount(surface_tilt, sliding_coefficient=1.97):
+    '''
+    Calculates the amount of snow that slides off of the surface of a module
+    following the model first described in [1] and later implemented with minor
+    improvements in SAM [2].
+
+    Parameters
+    ----------
+    surface_tilt : numeric
+        Surface tilt angles in decimal degrees. Tilt must be >=0 and
+        <=180. The tilt angle is defined as degrees from horizontal
+        (e.g. surface facing up = 0, surface facing horizon = 90).
+
+
+    sliding coefficient : numeric
+        Another empirically determined coefficient used in [1]. It determines
+        how much snow slides off of the panel if a sliding event occurs.
+
+    Returns
+    ----------
+    slide_amount : numeric
+        The amount of snow that slides off of the panel
+        in tenths of the panel area at each time step.
+
+    References
+    ----------
+    [1] Marion, B.; Schaefer, R.; Caine, H.; Sanchez, G. (2013).
+    “Measured and modeled photovoltaic system energy losses from snow for
+    Colorado and Wisconsin locations.” Solar Energy 97; pp.112-121.
+    [2] Ryberg, D; Freeman, J. "Integration, Validation, and Application
+    of a PV Snow Coverage Model in SAM" (2017) NREL Technical Report
+    '''
+
+    tilt_radians = np.radians(surface_tilt)
+    slide_amount = sliding_coefficient * np.sin(tilt_radians)
```

Comment:
slide_amount = (sliding_coefficient / 10) * np.sin(tilt_radians) * time_step (in hours).
I favor putting /10 in the calculation to maintain the meaning of slide_coefficient=1.97 in the references.